docs/command-line-interface/topaz-cli/authorizer.mdx

```diff
@@ -0,0 +1,259 @@
+
+```sh
+Usage: topaz authorizer (az) <command> [flags]
+
+authorizer commands
+
+Commands:
+  authorizer (az) eval             evaluate policy decision
+  authorizer (az) query            execute query
+  authorizer (az) decisiontree     get decision tree
+  authorizer (az) get-policy       get policy
+  authorizer (az) list-policies    list policies
+  authorizer (az) test             execute authorizer assertions
+
+Flags:
+  -h, --help        Show context-sensitive help.
+  -N, --no-check    disable local container status check ($TOPAZ_NO_CHECK)
+  -L, --log         log level
+```
+
+## Listing policies
+
+To list the policies that are loaded into the authorizer, use the `list-policies` subcommand:
+
+```sh
+topaz authorizer list-policies '{ "field_mask": "id" }' --insecure
+{
+  "result": [
+    {
+      "id": "todo/github/workspace/content/src/policies/rebac.check.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.DELETE.todos.__id.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.GET.todos.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.GET.users.__userID.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.POST.todos.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.PUT.todos.__id.rego"
+    },
+    {
+      "id": "todo/github/workspace/content/src/policies/todoApp.common.rego"
+    }
+  ]
+}
+```
+
+## Getting a policy
+
+To get the raw policy output for one of the modules, use the get-policy subcommand:
+
+```sh
+topaz az get-policy '{ "id": "todo/github/workspace/content/src/policies/todoApp.GET.todos.rego", "field_mask": "id,raw" }' --insecure
+{
+  "result": {
+    "id": "todo/github/workspace/content/src/policies/todoApp.GET.todos.rego",
+    "raw": "package todoApp.GET.todos\n\n# This policy determines whether the user can view all todos\n\ndefault allowed = true\n"
+  }
+}
+```
+
+## Evaluating a decision (`is` API call)
+
+To evaluate a decision using the `is` API call, use the `topaz authorizer eval` command. Note that the request requires an [identity context](/docs/authorizer-guide/identity-context.mdx) and a [policy context](/docs/authorizer-guide/policy-context.mdx). It may also contain an optional resource context.
+
+The policy context requires both an array of decisions to evaluate, as well as the path of the policy module to execute:
+
+```sh
+topaz authorizer eval '{
+  "identity_context": {
+    "type": "IDENTITY_TYPE_SUB",
+    "identity": "rick@the-citadel.com"
+  },
+  "policy_context": {
+    "decisions": [
+      "allowed"
+    ],
+    "path": "todoApp.GET.todos"
+  }
+}' --insecure
+{
+  "decisions":  [
+    {
+      "decision":  "allowed",
+      "is":  true
+    }
+  ]
+}
+```
+
+To print out a template for the `eval` command parameter, use the following command:
+
+```sh
+topaz authorizer eval -t
```

Comment:
I would use --template instead of the short version, to make it more apparent, easy to miss.